src/app/Fake.Tools.Octo/Octo.fs

```diff
@@ -0,0 +1,269 @@
+    Packages                : string list
+
+    /// A folder containing NuGet packages from which we should get versions
+    PackagesFolder          : string option
+
+    /// Release Notes for the new release
+    ReleaseNotes            : string
+
+    /// Path to a file that contains Release Notes for the new release
+    ReleaseNotesFile        : string
+
+    /// If a release with the version number already exists, ignore it
+    IgnoreExisting          : bool
+
+    /// Channel to use for the new release
+    Channel                 : string option
+
+    /// Ignore package version matching rules
+    IgnoreChannelRules      : bool }
+
+/// Options for deploying a release to an environment
+type DeployReleaseOptions = {
+    /// Name of the project
+    Project                     : string
+
+    /// Environment to deploy to
+    DeployTo                    : string
+
+    /// Version number of the release to deploy; Specify "latest" for
+    /// the latest release
+    Version                     : string
+
+    /// If a project is configured to skip packages with already-installed
+    /// versions, override this setting to force re-deployment
+    Force                       : bool
+
+    /// Whether to wait synchronously for deployment to finish
+    WaitForDeployment           : bool
+
+    /// Don't print the raw log of failed tasks
+    NoRawLog                    : bool
+
+    /// Show progress of the deployment.
+    /// (Sets --waitfordeployment and --norawlog to true.)
+    Progress                    : bool
+
+    /// Specifies maximum time that deployment can take
+    /// (default: 10 minutes)
+    DeploymentTimeout           : TimeSpan option
+
+    /// Specifies how much time should elapse between deployment status
+    /// checks (default: 10 seconds)
+    DeploymentCheckSleepCycle   : TimeSpan option
+
+    /// A comma-separated list of machine names to target in the
+    /// deployed environment. If not specified, all machines in
+    /// the environment will be considered.
+    SpecificMachines            : string option }
+
+/// Options for deleting a range of releases in a project
+type DeleteReleaseOptions = {
+    /// Name of the project
+    Project     : string
+
+    /// Minimum (inclusive) version number for the range of versions to delete
+    MinVersion  : string
+
+    /// Maximum (inclusive) version number for the range of versions to delete
+    MaxVersion  : string }
+
+type PushOptions = {
+    // paths to one or more packages to push to the server
+    Packages : string list 
+    /// if the package already exists, should this package overwrite it?
+    ReplaceExisting : bool 
+}
+
+/// Option type for selecting one command
+type OctoCommand = 
+| CreateRelease of CreateReleaseOptions * DeployReleaseOptions option
+| DeployRelease of DeployReleaseOptions
+| DeleteRelease of DeleteReleaseOptions
+| ListEnvironments
+| Push of PushOptions
+
+/// Complete Octo.exe CLI params
+[<CLIMutable>]
```

Comment:
Generally we remove `CLIMutable` is that required here?

Reply:
I do not think so. Will double-check and remove if not.